hatch/src/generators/tup/platform.rs

```diff
@@ -0,0 +1,54 @@
+pub mod linux {
+    pub fn name() -> String {
+        String::from("linux.tup")
+    }
```

Comment:
I think these name functions are unused, but we might want to use. Them later. I honestly think we should just inline the strings, but that is just me.